signature/internal/signaturetest/algorithm.go

```diff
@@ -1,3 +1,5 @@
+// Package signaturetest includes variables and functions for signature
```

Comment:
If it is for internal test purposes only, is it possible to name `algorithm.go` to `algorithm_test.go`, `signer.go` to `signer_test.go`?

Reply:
I tried it. However, it looks like variables and functions in a file endded with "_test.go" cannot be used out side the packge. Those functions are shared by signature/jws and signature/cose packages.

Reply:
I see. Can we also write unit tests for those "test functions"? It will also impact the coverage.

Reply:
Added test cases.